tests/test_flip.py

```diff
@@ -0,0 +1,65 @@
+from visumorph.flip import flip
+from visumorph import load_image
+import numpy as np
+import pytest
+import os
+
+# Create a directory to store the test results
+RESULTS_DIR = 'tests/img/results/flip_results'
+os.makedirs(RESULTS_DIR, exist_ok=True)
+
+
+def test_flip_vertically():
+    img = load_image('tests/img/raw/meme.jpg')
+    flipped_img = flip(img, v=1)
+
+    assert flipped_img.get_dimensions() == img.get_dimensions()
+
+    # Check if the top row of pixels is the same in both images (since it's flipped vertically)
+    assert np.array_equal(flipped_img.image[0, :], img.image[-1, :])
+
+    # Save the flipped image to the results directory
+    flipped_img.save(os.path.join(RESULTS_DIR, 'flipped_vertically.jpg'))
```

Comment:
Will the saved image be tested afterwards?

Reply:
No, in the end I think I was exporting them for me more than to test them. Removed in later update.

Reply:
Yeah just saw that in the latest commit, I agree storing the flipped image and use them to compare the function output would be a better approach